crowbar_framework/lib/crowbar/lock.rb

```diff
@@ -16,33 +16,14 @@
 #
 
 module Crowbar
-  class Lock
-    attr_accessor :logger, :path, :name
-    attr_reader :path, :name, :file
+  module Lock
+    autoload :Base,
```

Comment:
> Because per default rails won't autoload the lib folder, this is currently just worked around by us but i want to drop that workaround.

I still don't understand why we don't fix another way. Instead of

```
    config.autoload_paths += %W(
      #{config.root}/lib
    )
```

we could have

```
    config.autoload_paths += %W(
      #{config.root}/lib/**/
    )
```